app.env.example

```diff
@@ -15,4 +15,12 @@ MONGO_USER=hs_auth
 MONGO_PASSWORD=password123
 
 JWT_SECRET="very secret phrase"
+
+# Only required if the SenGrid client is used in wire.go
```

Comment:
```suggestion
# Only required if the SendGrid client is used in wire.go
```

Reply:
We may want to mention the email provider is configured via the `EmailDeliveryProvider` config property instead